WebServer/microservices/streamer/Dockerfile

```diff
@@ -0,0 +1,64 @@
+FROM buildpack-deps:stretch
+
+LABEL maintainer="Sebastian Ramirez <tiangolo@gmail.com>"
```

Comment:
??

Reply:
Har stort set kopieret hans Dockerfil. Tænker han har skrevet labelet i filen for at få lidt credit. Men vi kan sagtens slette det.